src/main.rs

```diff
@@ -139,7 +139,7 @@ fn main() -> Result<(), Box<dyn Error>> {
         if options.truncate {
             writeln!(
                 std::io::stdout(),
-                "TRUNCATE TABLE\n  {}\n;",
+                "TRUNCATE TABLE\n  {}\nCASCADE;",
```

Comment:
Whitespace doesn't _technically_ matter here but if anyone ever needs to edit a dump file, they may wish these were on one line.
```suggestion
                "TRUNCATE TABLE {} CASCADE;\n",
```

Reply:
Ignore me. I thought this was once per table, but it's a list of tables right?

Reply:
Yup, it's a list of tables